deploy/demo/env.yaml

```diff
@@ -11,53 +11,58 @@ spec:
-            valueFrom:
-              secretKeyRef:
-                name: commonurl-demo
-                key: FDK_REFERENCE_DATA_BASE_URI
-          - name: ELASTIC_HOST
-            valueFrom:
-              secretKeyRef:
-                name: fdk-reports-store-demo
-                key: FDK_REPORTS_STORE_HOST
-          - name: ELASTIC_PORT
-            valueFrom:
-              secretKeyRef:
-                name: fdk-reports-store-demo
-                key: FDK_REPORTS_STORE_PORT
-          - name: FDK_BASE
-            valueFrom:
-              secretKeyRef:
-                name: commonurl-demo
-                key: FDK_BASE_URI
-          - name: SPARQL_BASE
-            valueFrom:
-              secretKeyRef:
-                name: commonurl-demo
-                key: FDK_SPARQL_SERVICE_QUERY_URI
-          - name: API_KEY
-            valueFrom:
-             secretKeyRef:
-               name: common-demo
-               key: FDK_REPORTS_BFF_API_KEY
+            - name: CONCEPT_HARVESTER_URL
+              valueFrom:
+                secretKeyRef:
+                  name: commonurl-demo
+                  key: FDK_CONCEPT_HARVESTER_URI
+            - name: DATASET_HARVESTER_URL
+              valueFrom:
+                secretKeyRef:
+                  name: commonurl-demo
+                  key: FDK_DATASET_HARVESTER_URI
+            - name: DATASERVICE_HARVESTER_URL
+              valueFrom:
+                secretKeyRef:
+                  name: commonurl-demo
+                  key: FDK_DATASERVICE_HARVESTER_URI
+            - name: INFORMATIONMODELS_HARVESTER_URL
+              valueFrom:
+                secretKeyRef:
+                  name: commonurl-demo
+                  key: FDK_FULLTEXT_SEARCH_BASE_URI
+            - name: FDK_REFERENCE_DATA_URL
+              valueFrom:
+                secretKeyRef:
+                  name: commonurl-demo
+                  key: FDK_REFERENCE_DATA_BASE_URI
+            - name: ELASTIC_HOST
+              valueFrom:
+                secretKeyRef:
+                  name: fdk-reports-store-demo
+                  key: FDK_REPORTS_STORE_HOST
+            - name: ELASTIC_PORT
+              valueFrom:
+                secretKeyRef:
+                  name: fdk-reports-store-demo
+                  key: FDK_REPORTS_STORE_PORT
+            - name: FDK_BASE
+              valueFrom:
+                secretKeyRef:
+                  name: commonurl-demo
+                  key: FDK_BASE_URI
+            - name: SPARQL_BASE
+              valueFrom:
+                secretKeyRef:
+                  name: commonurl-demo
+                  key: FDK_SPARQL_SERVICE_QUERY_URI
+            - name: API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: common-demo
+                  key: FDK_REPORTS_BFF_API_KEY
+            - name: CORS_ORIGIN_PATTERNS
+              valueFrom:
+                secretKeyRef:
+                  name: fdk-reports-store-demo
```

Comment:
dette er vel en secret mest tenkt for medfølgende elastic. Den skal _snart_ stenges når indeksen flyttes til elastic-clusteret, hadde sikkert vært greit å bare kunne slette hele secreten da. Elelr @valosnah?

tldr: sikkert bedre å ha det i en egen secret for fdk-reports-bff